gard_owl_ingest/analysis/mondo_mapping_status.py

```diff
@@ -0,0 +1,48 @@
+"""Mapping status between GARD and Mondo"""
+import os
+from pathlib import Path
+from typing import List, Set
+
+import pandas as pd
+
+CURIE = str
+ANALYSIS_DIR = Path(os.path.dirname(os.path.abspath(__file__)))
+SRC_DIR = ANALYSIS_DIR.parent
+PROJECT_DIR = SRC_DIR.parent
+DATA_DIR = PROJECT_DIR / 'data'
+TMP_DIR = PROJECT_DIR / 'tmp'
+DATASOURCE_CSV = DATA_DIR / 'GARD_disease_list.csv'
+MONDO_SSSOM_TSV = TMP_DIR / 'mondo.sssom.tsv'
+
+
+def gard_mondo_mapping_status():
```

Comment:
@matentzn Feel free to merge this or we can also just close this, as it is ad hoc.

Refer to my comment in [the issue](https://github.com/monarch-initiative/gard/issues/2#issuecomment-1499785254) for discussion.